pom.xml

```diff
@@ -15,17 +27,11 @@
   </properties>
 
   <dependencies>
-<!-- https://mvnrepository.com/artifact/junit/junit -->
-<!-- moved junit testing to another project 1/7/18 -->
-<!-- 
-	<dependency>
-	    <groupId>junit</groupId>
-	    <artifactId>junit</artifactId>
-	    <version>4.12</version>
-	    <scope>test</scope>
-	</dependency>
--->
-
-
+      <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+          <version>RELEASE</version>
```

Comment:
Specify the junit version here or set to use the latest release following this stackoverflow answer https://stackoverflow.com/questions/30571/how-do-i-tell-maven-to-use-the-latest-version-of-a-dependency